src/colp/components/manual_correction_report.py

```diff
@@ -0,0 +1,42 @@
+import streamlit as st
+import pandas as pd
+
+
+class ManualCorrection:
+    def __init__(self, data):
+        self.modifications_applied = data["modifications_applied"]
+        self.modifications_not_applied = data["modifications_not_applied"]
+
+    def __call__(self):
+        st.subheader("Manual Correction Report")
+
+        if (
+            self.modifications_applied is None
```

Comment:
This can throw a bug is one table is none and the other is populated. Know this isn't very likely but I do think we should have a way to handle it